### ISSUE TITLE
[Feature] OCR

### ISSUE THREAD

**Issue by Atrate** [Aman-zishan/DocScanner#3]
When edge detection is implemented, a nice to have feature would probably be some kind of OCR (optical character recognition) solution.

**Comment by Aman-zishan** [comment]
I'm looking forward to the same...are u interested in collaborating in this project

**Comment by Atrate** [comment]
I am not terribly experienced as an Android developer but I can help with stuff like code cleanup and documentation

**Comment by Prakhar314** [comment]
What will you be using for edge detection, gray scale, ocr, other image processing tasks?
I thought of using [opencv](https://pub.dev/packages/opencv), but the flutter plugin only works for Android right now.

**Comment by Atrate** [comment]
Best OCR would probably be obtainable with the help of Tesseract

**Comment by Jeewantha-Madubashana** [comment]
Can i contribute?

**Comment by Aman-zishan** [comment]
> What will you be using for edge detection, gray scale, ocr, other image processing tasks?
> I thought of using [opencv](https://pub.dev/packages/opencv), but the flutter plugin only works for Android right now.

if we want to perform ocr we could use the API from the other webapp(textextractor) which is based on tesseract. And about the filters i was thinking of opencv too.

**Comment by Aman-zishan** [comment]
> Can i contribute?

Please do mention on which issue you would be working on

**Comment by pushpulldecoder** [comment]
Won't it increase the size of the application if local ocr is implemented?

**Comment by Aman-zishan** [comment]
Yes it will, that why we were planning  to implement through google cloud vision API /tesseract API or simply using [textextractor2](http://textextractor2.herokuapp.com/api/v1/) API
